### DISCUSSION TITLE
Setting OpenId Auth besides Keycloak

### DISCUSSION THREAD

**Discussion by fjcpwithnocorpaccount** [ApicurioArchive/apicurio-studio#2212] [General]
Hello all! 

I´ve checked the answer given by @EricWittmann regaring this topic:

https://github.com/Apicurio/apicurio-studio/discussions/2039

We consider that it won´t provide the full package of functionality (integration with gitlab, github) but basic will probably there.

However, we´ve detected that setting an URL like:

https://foo.organization.com/IDP/.well-known/openid-configuration

in variable: APICURIO_KC_AUTH_URL it does not work, because the result URL to retrieve the openid-configuration will be composed as:

APICURIO_KC_AUTH_URL + /realm/ + APICURIO_KC_REALM

is there a way to disable this and avoid the realm attachement?

Thanks a lot and regards!

**Comment by EricWittmann** [comment]
Sorry about that - Studio has been on the back burner as a priority for awhile, although that's changing.  We're working on restarting this project with a 1.0 of Studio that is a bit of a rewrite.  So the good news is that we'll be addressing things like this.  The bad news is that some functionality from the 0.2 edition of Studio (including integration with e.g. GitHub) will not make it into 1.0.
